### PR TITLE
Integrate the basecurve tool into the main build

### PR DESCRIPTION
**Pull request by cryptomilk**
This is optional and can be enabled with:

-DBUILD_BASECURVE_TOOLS=ON

### CODE REVIEW

**Comment by cryptomilk** [comment]
Ping

**Comment by cryptomilk** [comment]
@TurboGit

File: tools/basecurve/CMakeLists.txt [hunk 1]
```diff
@@ -3,18 +3,18 @@ find_package(OpenMP)
 find_package(Exiv2 REQUIRED)
 include_directories(${EXIV2_INCLUDE_DIR})
 
-add_executable(darktable-curve-tool darktable-curve-tool.c exif-wrapper.cpp)
```
Comment (TurboGit) [inline tools/basecurve/CMakeLists.txt]: this is wrong, please do not rename as the curve-tool can handle basecurve and tonecurve.

**Comment by cryptomilk** [comment]
I will fix it.

**Comment by cryptomilk** [comment]
I've updated the branch ...

**Comment by TurboGit** [comment]
Thanks working fine to me.

**Pull request merged**